bitbots_body_behavior/config/body_behavior.yaml

```diff
@@ -195,6 +195,21 @@ behavior:
     # estimated kick length when estimating kick cost
     kick_cost_kick_length: 1.5
 
+    # parameters for time_to_ball estimation
+    # divider of how often the time to ball is updated depending on update rate of the behavior
+    # example: (125 = 1 per second, 250 = 1 per 2 seconds)
+    time_to_ball_divider: 125
```

Comment:
I would run this with more than 1 herz. Otherwise, we might wait for more than a second until we start to go to the ball in e.g. pass situation.

Reply:
increased to 5 Hz by setting divider to 25